quickjs.c

```diff
@@ -49870,6 +49870,11 @@ static JSValue js_date_constructor(JSContext *ctx, JSValueConst new_target,
     return rv;
 }
 
+int JS_ToDate(JSContext *ctx, double *pres, JSValueConst val)
+{
+    return JS_ThisTimeValue(ctx, pres, val);
+}
```

Comment:
You can get the same result with `JS_ToFloat64(ctx, pres, val)`.

The difference is your proposed wrapper is thinner and only works for actual `Date` objects. The name is a bit misleading too, I would have used `JS_DateTimeValue` or something similar, but I have yet to come up with a consistent naming scheme for such utility functions.

Reply:
I somehow missed that `JS_ToFloat64()` did the same thing `Date` objects! I'm more than happy to just do that.

Reply:
For reference: https://github.com/holepunchto/libqjs/commit/a37fd4f993d8b70b8c108adfc11df42f8b06b217